reader.go

```diff
@@ -90,9 +91,11 @@ func (rdr *LogLineReader) ReadLines() error {
 			rdr.linesRead.Inc(1)
 			rdr.mu.Lock()
 			if full := rdr.b.Add(LogLine{line, currentLogTime}); full {
+				rdr.batchFillTime.UpdateSince(now)
```

Comment:
Could we use `time.Now()` directly as the argument of .UpdateSince without a variable ?